src/coreclr/jit/emitxarch.cpp

```diff
@@ -18543,23 +18549,32 @@ emitter::insExecutionCharacteristics emitter::getInsExecutionCharacteristics(ins
         case INS_cvtsi2sd64:
         case INS_cvtsi2ss64:
         case INS_vcvtsd2usi:
-        case INS_vcvttsd2usi:
-        case INS_vcvtusi2sd32:
-        case INS_vcvtusi2sd64:
         case INS_vcvtusi2ss32:
         case INS_vcvtusi2ss64:
+        case INS_vcvttsd2usi:
```

Comment:
Why is `vcvtss2usi` being split into `32/64` but `vcvttsd2usi` is not?

Reply:
We have not yet optimized ```double -> uint```. Hence, this was left untouched. When we optimize ```double -> uint```, this will also be split into 2 i.e. ```vcvttsd2usi32 / vcvttsd2usi64```

Reply:
I think we should either split it in this PR or remove it until we optimize the double the unit case. Probably the latter as it's not tested.

Reply:
> or remove it until we optimize the double the unit case

The instruction is already supported today for explicit intrinsic usage, this was just moved (likely inadvertently) as part of moving `vcvttsd2usi32`

The actual change is fine and it makes sense to not split both if we're not doing the other changes for `cvttsd2usi` in this pr